interface/forms/aftercare_plan/new.php

```diff
@@ -1,20 +1,22 @@
 <?php
 
-/**
```

Comment:
Dumb, I know, but I think the double asterisks should stay for doc-blcks

Reply:
In phpstorm, for the license block, it now uses a single asterisk. It does not treat it as doc block.